lib/node_modules/@stdlib/math/base/complex/divide/test/test.js

```diff
@@ -0,0 +1,283 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var isinfinite = require( '@stdlib/math/base/assert/is-infinite' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var toBinaryString = require( '@stdlib/math/base/utils/float64-to-binary-string' );
+var cdiv = require( './../lib' );
+
+
+// FIXTURES //
+
+var quotients = require( './fixtures/julia/quotients.json' );
+
+
+// HELPERS
```

Comment:
Ohh, maybe that was the issue. I only counted bits correct *before* a bit became incorrect. That is, `00101` and `01101` have only one bit correct because presumably the least significant bits aren't meaningfully correct. Like 10000015 and 16 have a much bigger problem than one incorrect bit. I have no strong feelings on what's best there. It's just a comparison.